src/test/resources/application-test.yml

```diff
@@ -23,6 +23,10 @@ spring:
       resourceserver:
         jwt:
           issuer-uri:
+  session:
+    store-type: jdbc
+    jdbc:
+      initialize-schema: embedded
```

Comment:
The unit tests do not run DB migrations so need to tell it to run in memory (which is what the above is doing).